src/content/docs/en/reference/feature-flags/experimental-client-prerender.mdx

```diff
@@ -0,0 +1,42 @@
+---
+title: Experimental client prerendering
+sidebar:
+  label: Client prerendering
+i18n: true
```

Comment:
Should these be `i18nReady` in all of these files?

Reply:
Yes! Thank you!